waku/waku_test.go

```diff
@@ -894,10 +916,18 @@ func TestSymmetricSendKeyMismatch(t *testing.T) {
 	InitSingleTest()
 
 	w := New(nil, nil)
-	defer w.SetMinimumPoW(common.DefaultMinimumPoW, false)  // nolint: errcheck
-	defer w.SetMaxMessageSize(common.DefaultMaxMessageSize) // nolint: errcheck
-	w.Start(nil)                                            // nolint: errcheck
-	defer w.Stop()                                          // nolint: errcheck
+	if err := w.Start(nil); err != nil {
+		t.Errorf("failed to start waku: '%s'", err)
+	}
+	defer func() {
+		handleError(t, w.SetMinimumPoW(common.DefaultMinimumPoW, false))
+	}()
+	defer func() {
+		handleError(t, w.SetMaxMessageSize(common.DefaultMaxMessageSize))
+	}()
+	defer func() {
+		handleError(t, w.Stop())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)